config/routes.rb

```diff
@@ -11,4 +11,13 @@
 
   # Defines the root path route ("/")
   # root "posts#index"
+  namespace :api do
+    namespace :v1 do
+      resources :users, only: [:show] do
+        resource :schedule, only: [:show, :update]
```

Comment:
Is your plan to show a list of users and then click on each user to see their schedule(s)? If so I think this is ok - but if you're trying to show schedules in the admin index this might be a problem because you don't have a top level schedules endpoint.

I think you should consider whether you might want a general schedules index that takes a `user_id` as a param rather than encoding it in the route. I'm not sure what your front end requirements are, but always needing a user id to make any schedule requests is pretty restrictive for an admin view.

Another option is to namespace a schedule endpoint inside `admin` that would allow for larger queries and then have a user scoped one like this in the public or users namespace.

Just things to think about.